packages/forms/src/components/field.gts

```diff
@@ -103,12 +123,31 @@ class Field<
     return get(this.args.formData, this.args.name);
   }
 
+  @action
```

Comment:
`@action` decorator is not needed here as this method is not called from a template.